File: bin/advice.py
#! /usr/bin/env python
# Simple advice client for https://api.adviceslip.com

import requests

API_URL = 'https://api.adviceslip.com/advice'

response = requests.get(API_URL).json()

if ('slip' in response):
    id = response['slip']['id']
    advice = response['slip']['advice']

    print(f'[{id}] {advice}')
else:
    print(f'No response from {API_URL}')

File: bin/advice.js
#! /usr/bin/env node

async function getAdvice() {
    let data = await (
            await fetch('https://api.adviceslip.com/advice')
        ).json();

    return {
        id: data.slip.id,
        advice: data.slip.advice,
    };
}

async function main() {
    let advice = await getAdvice();
    console.log(`[${advice.id}] ${advice.advice}`);
}

main();
